packages/cli-packages/storybook-vite/src/__tests__/storybookConfigFixtures.test.ts

```diff
@@ -0,0 +1,40 @@
+import fs from 'fs'
+
+import { describe, it, expect } from 'vitest'
+
+describe('storybook config file fixtures', () => {
+  it('main.js', () => {
+    const mainTemplatePath = `${__dirname}/../commands/templates/main.ts.template`
+    console.log('mainTemplatePath', mainTemplatePath)
+    const mainTemplate = fs.readFileSync(mainTemplatePath, { encoding: 'utf8' })
+    expect(mainTemplate)
+      .toEqual(`import type { StorybookConfig } from 'storybook-framework-redwoodjs-vite'
```

Comment:
I'd probably make this a snapshot test, just because it's easier to update if/when the template changes. But good to leave like this too

Reply:
yeah fair point — did it this way because the content is so short as of now that it's not a lot to have it inline. unless you feel strongly, i'd prefer to keep it this way and we can convert to snapshot later on if the content grows

Reply:
As I said, I'm good with leaving it as it is.
But I do want to also mention that you could use an inline snapshot if you wanted 🙂 That way you can exactly see what is expected, but you also don't have to manually update it if it ever changes

Reply:
OOOH yes good callout, totally slipped my mind that that's an option — updated to do exactly that :)